package.json

```diff
@@ -82,8 +82,8 @@
     "react-dom": "^18.2.0",
     "react-dropzone": "^14.2.3",
     "react-helmet": "^6.1.0",
-    "react-hook-form": "^7.37.0",
-    "react-i18next": "^11.18.6",
+    "react-hook-form": "^7.38.0",
+    "react-i18next": "^12.0.0",
```

Comment:
Types of the `t` function have changed: https://github.com/i18next/react-i18next/blob/master/CHANGELOG.md

Scroll down the Diff here on Github:

<img width="1466" alt="image" src="https://user-images.githubusercontent.com/7814926/197455837-77d287ac-d929-4b8c-93de-17b29c82e224.png">

Reply:
The dependency bot has now also updated `gatsby-plugin-react-i18next`. I guess they fixed the types, since there are no tests failing now.